chatbot.html

```diff
@@ -358,39 +358,319 @@ <h2>DIETBOT</h2>
+              If it is in your plan that you do not need such a large amount of food, then you can reduce the portion of what you buy. Always make a conscious effort to serve food in smaller portions. <br>
+              Even in restaurants and if you do happen to have more than what you can finish on the table, there is never a problem to take the food back as a take-away to finish it back home.</p>
+          </div>
+        </div>
+
+        <div id="ai">
+          <h1>What is Artificial Intelligence?</h1>
+          <p>Artificial intelligence is a theory and development of computer systems to be able to perform tasks normally requiring human intelligence. </p>
+          <h2>How are we implementing Artificial Intelligence into our solution? </h2>
+          <h3>Current applications and implementations of AI technology: </h3>
+          <ul>
+            <li>Automation</li>
+            <li>Machine learning</li>
+            <li>Machine vision</li>
+            <li>Natural language processing(NLP)</li>
+            <li>Robotics</li>
+            <li>Self-driving cars</li>
+          </ul>
+          <h3>How are we implementing AI to solve our solution?</h3>
+          <p>We will leverage the use of AI to provide methods of what meals has the least carbon emissions so that our users will not spend and waste on food unneccessarily. </p>
+          <p>With this technology, less food will be wasted and the lesser amount of carbon emissions will also reduce the rate of greenhouse gases being trapped in the atmosphere and as well as minimising climate change as much as possible, with the technology of our chatbot.</p>
+          <p>Scroll down to find out more what we intend to do with our</p>
+        </div>
     </div>
+    <!-- Opening tabs function  -->
+    <script>
+    //redirects when user clicks on one of the slides and opens the tab for them
+    function gototab(redirect_id) {
+      location.replace(redirect_id);
+      opentab(event, redirect_id);
+    }
+    //sets this tab open by default when user first loads their browser
+    document.getElementById("defaultTab").click();
+    //main tab function
+    function opentab(evt, id_name) {
+      var i, tabcontent, id_name;
+
+      // Get all elements with class="tabcontent" and hide them
+      tabcontent = document.getElementsByClassName("tabcontent");
+      for (i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = "none";
+        
+      }
+
+      // Get all elements with class="tablinks" and remove the class "active"
+      let tablinks = document.getElementsByClassName("tablinks");
+      for (i = 0; i < tablinks.length; i++) {
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
+      }
+      
+      // Show the current tab, and add an "active" class to the button that opened the tab
+      document.querySelector(id_name).style.display = "block";
+      evt.currentTarget.className += " active";
+
+
+    }
+    </script>
```

Comment:
Above here, it is my JavaScript program that maintains all the tab functions and its overall interactivity to the users.